modules/apiserversource-kn.adoc

```diff
@@ -83,16 +83,18 @@ Conditions:
 
 .Verification
 
-You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.
 
-. Get the pods:
+. To view the events in a web browser, open the link returned by the following command:
 +
 [source,terminal]
 ----
-$ oc get pods
+$ kn service describe event-display -o url
 ----
 
-. View the message dumper function logs for the pods:
+image::serverless-apiserversource-eventdisplay-gui.png[Example visualization of ApiServerSource event]
+
+. Alternatively, to see the logs in the terminal, view the event-display logs for the pods by entering the following command:
```

Comment:
This (and a couple lines after this) seems to cover the same ground as the lines in the late 80s above. Can they be consolidated?

Reply:
I don't get your proposal, would you kindly specify it more clearly?

My understanding:

Paragraph at 86 describes there are 2 ways of verification: in-browser, or logs

Lines 88-95 shows the first way of using the web browser to do the verification.

Lines 97-135 shows the second method - the logs.

Reply:
The words for verifying via the even logs in lines 86 and 97 are almost identicle. nothing new is added to give value to the reader. My suggestion is to move the command from Line 86 to Line 97.

Reply:
The procedure from 88-95 shows hot to get the URL and how that web page should look like (image attached).

The second procedure 97-135 shows different command to display logs in the terminal. It's entirely different.

Reply:
I don't see a clear way on how I could fold those procedures into one, while keeping the current meaning.

Please, maybe you could suggest a wording you have in mind...

Reply:
@cardil At the least, you could change the numbered list to unordered, as the two methods are not performed sequentially -- I should think most users would do one of the other and not both.